source/ui/Components/Memory/HeaderButton.qml

```diff
@@ -0,0 +1,98 @@
+/* C++ Assembler Interpreter
+* Copyright (C) 2016 Chair of Computer Architecture
+* at Technical University of Munich
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.*/
+
+import QtQuick 2.6
+import QtQuick.Controls 1.5
+import QtQuick.Controls.Styles 1.4
+import QtGraphicalEffects 1.0
+
+import Theme 1.0
+
+// The ComboBox above each row the user can either choose the number
+// of bits or the numerical representation of a memory cell depending
+// on the column.
+ComboBox {
+  id: root
+  height: root.height
+  anchors {
+    top: parent.top
+    bottom: parent.bottom
+    left: parent.left
+    right: remove.right
+    rightMargin: {
+      if (currentRole === 'address') {
+        return 0;
+      } else {
+        return Theme.memory.header.remove.marginLeft;
+      }
+    }
+  }
+
+  style: ComboBoxStyle {
+    background: Rectangle {
+      color: Theme.memory.header.background
+    }
+    label: Text {
+      font.pixelSize: Theme.memory.header.fontSize
+      horizontalAlignment: Qt.AlignHCenter
+      verticalAlignment: Qt.AlignVCenter
+      text: (currentRole === 'address') ? 'Address' : 'Memory'
```

Comment:
On memory columns, this always says memory, regardless of format:
![memory](https://cloud.githubusercontent.com/assets/18463279/22174045/a964555a-dfd4-11e6-86bd-2174841714cf.png)
This way, you have to click on it in order to see the format type.
I think it would be nice to use the same selector as the registers, with the B, H, S, U labels.